cardano-integration/src/Test/Integration/Cardano/Process.hs

```diff
@@ -19,6 +22,15 @@ execCli = exec "cardano-cli"
 execCli' :: MonadIO m => [String] -> m (ExitCode, String, String)
 execCli' = exec' "cardano-cli"
 
+execMarlowe :: MonadIO m => [String] -> m String
+execMarlowe = exec "bash" . ("./marlowe-integration-tests/marlowe" :)
+
+execMarlowe_ :: MonadIO m => [String] -> m ()
+execMarlowe_ = void . execMarlowe
+
+execMarlowe' :: MonadIO m => [String] -> m (ExitCode, String, String)
+execMarlowe' = exec' "bash" . ("./marlowe-integration-tests/marlowe" :)
+
```

Comment:
Could we move these to a module in `marlowe-integration-tests`? They don't really belong in `cardano-integration` (feel free to expose `exec` and `exec'` from this module. Also, if you define these there, you can hide away some of the parameter passing. For example, if these were defined in `Language.Marlowe.Runtime.Integration.Common`, you could do something like this:

```hs
execMarlowe :: [String] -> Integration String
execMarlowe = exec "marlowe" <=< prepareCliArgs

execMarlowe_ :: [String] -> Integration ()
execMarlowe_ = void . execMarlowe

execMarlowe' :: [String] -> Integration (ExitCode, String, String)
execMarlowe' = exec' "marlowe" <=< prepareCliArgs

prepareCliArgs :: [String] -> Integration [String]
prepareCliArgs args = do
  MarloweRuntime{..} <- ask
  pure $ ["--marlowe-runtime-port", show port] <> args
```